src/NUnitFramework/framework/Attributes/IgnoreAttribute.cs

```diff
@@ -29,9 +29,7 @@
 namespace NUnit.Framework
 {
     /// <summary>
-    /// Attribute used to mark a test that is to be ignored.
-    /// Ignored tests result in a warning message when the
-    /// tests are run.
+    /// Marks a test as being ignored. Ignored tests result in a warning message when the tests are run.
```

Comment:
The other places we have also mentioned test fixture (and perhaps also assembly)